### PR TITLE
feat: add header component

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

**Review by Veryyapeee — DISMISSED** [review]
Menu is not responsive at all

**Comment by T0ffik** [comment]
> Menu is not responsive at all
Done. I think.

**Comment by T0ffik** [comment]
> Menu is not responsive at all

**Comment by T0ffik** [comment]
> Menu is not responsive at all 

Done. I think.

**Review by T0ffik — COMMENTED** [review]
Done. I think.

**Comment by Veryyapeee** [comment]
I think that we need some horizontal padding in this menu

File: src/components/header/Header.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { render } from '../../shared/dom';
+
+const template= (close, open) =>`
+    <div id="header">
```
Comment (Veryyapeee) [inline src/components/header/Header.js]: This should be a `<header/>` tag

File: src/components/header/_header.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
+@import '../../../styles/variables';
+
+#header{
```
Comment (Veryyapeee) [inline src/components/header/_header.scss]: Generally to whole styling here
Don't use ID's for styling, use class instead

File: src/components/header/_header.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+@media (max-width : 1316px) {
+  #header{
+    display: flex;
+    font-size: 10px;
+    justify-content: space-between;
+    align-items: center;
+    height: 80px;
+    padding-left: 10px;
```
Comment (Veryyapeee) [inline src/components/header/_header.scss]: you can replace it with
`padding: 0 10px;`

File: src/components/header/_header.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        display: none;
+        text-align: center;
+        padding-left: 25px;
+        padding-right: 25px;
+        #exit{
+          display: contents;
+          cursor: pointer;
+          border: none;
+          outline: none;
+          background-color: rgba(0, 0, 0, 0);
```
Comment (Veryyapeee) [inline src/components/header/_header.scss]: even if it's just a black color, this should be in variables, add it please

File: src/components/header/_header.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      }
+      .btns{
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+      ul{
+        display: none;
+        text-align: center;
+        padding-left: 25px;
```
Comment (Veryyapeee) [inline src/components/header/_header.scss]: `padding: 0 25px;`

File: src/components/header/_header.scss [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+        }
+        li{
+          margin: 0;
+        }
+      }
+    }
+    .menu-active{
+      position: absolute;
+      display: block !important;
+      background-color: #B9B9B7;
```
Comment (Veryyapeee) [inline src/components/header/_header.scss]: This should be in variables.scss

**Review by Veryyapeee — DISMISSED** [review]
check the comments

File: src/components/header/Header.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+      <div class="navBar">
+        <ul id="menu" class="menu">
+          <li>Movie</li>
+          <li>TV Series</li>
+          <li>Search</li>
+          <li>Favorite</li>
+          <button id="exit" class="exit"><li>&times;</li></button>
+        </ul>
+        <div class="btns">
+          <button id="icon" class="icon"><img src="/static/img/menuIcon.png" alt="menu"></button>
```
Comment (Veryyapeee) [inline src/components/header/Header.js]: Alt should be meaningfull, it's for accesibility reasons, so it should be something like, toggleMenuIcon
Minor

File: src/components/header/Header.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
+import { render } from '../../shared/dom';
+
+const template= (close, open) =>`
```
Comment (Veryyapeee) [inline src/components/header/Header.js]: You are not using this close and open functions,

Convert this back to be an const instead od function, just plain html :p

File: src/components/header/Header.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    <header>
+      <img src="/static/img/logo.png" alt="logo" class="logo">
+      <div class="navBar">
+        <ul id="menu" class="menu">
+          <li>Movie</li>
```
Comment (Veryyapeee) [inline src/components/header/Header.js:10]: Add TODO that in the future we need to connect these elements with redirect function

**Review by Veryyapeee — DISMISSED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**